actors/src/test/kotlin/kts/actors/actionsTest.kt

```diff
@@ -58,7 +59,7 @@ class ActionsTest {
   }
 
   @Test
-  fun `should chain actions into sequences`() {
+  fun `action then action should chain actions into sequences`() {
```

Comment:
Personal preference: tests names starting with `should`. E.g.: `should chain actions into sequences with then infix function`.